site/content/en/reference/live/_index.md

```diff
@@ -20,4 +20,129 @@ description: >
 
 Live contains the next-generation versions of apply related commands for
 deploying local configuration packages to a cluster.
+
+### How to Upgrade to the Next Generation Inventory Object
+
+#### What is an Inventory Object
+
+An inventory object is the automatically generated object which keeps track
+of the set of objects applied together. The current inventory object type
```

Comment:
Goes back to comment above, these docs need to be future looking, so when they're released the 'current way' will be using ResourceGroup and the 'old way' will be using ConfigMap.